strangerstudios/sitewide-sales
# Sale content refactor

### All Submissions:






### Changes proposed in this Pull Request:
Refactored blocks by restructuring code files. Incorporated new sale content block through modifying 'includes/blocks.php' and 'webpack.config'. Introduced a separate folder in 'sources' for new block files. Transferred code from 'blocks/block.js' and 'blocks/block.php' to individual files within the new block folder.



### How to test the changes in this Pull Request:

1. Make sure there is a sitewide sale active
2. insert sale content block in the page editor
3. Choose different sale period settings
4. View the block on the frontend




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

- Modifcations to blocks.php and webpack.config.js
- New src/sale-content folder with block files
